src/logging.cpp

```diff
@@ -84,17 +83,18 @@ init(bool console, quill::LogLevel log_level, bool structured) {
     }
 
     // Initialize file handler
-    quill::Handler* file_handler;
+    std::shared_ptr<quill::Handler> file_handler;
 #if DEBUG()
     (void)structured; // Keep the compiler from complaining
 
     // Rotate through file handlers to save space
-    file_handler = quill::rotating_file_handler(
-        LOG_FILE,
-        "a",             // append
-        1024 * 1024 / 2, // 512 KB
-        5                // 5 backups
-    );
+    file_handler = quill::rotating_file_handler(LOG_FILE, []() {
+        quill::RotatingFileHandlerConfig cfg;
+        cfg.set_rotation_max_file_size(1024 * 1024 / 2); // 512 KB
+        cfg.set_max_backup_files(5);                     // 5 backups
+        cfg.set_overwrite_rolled_files(true);            // append
+        return cfg;
+    }());
 #else
     if (structured) {
```

Comment:
https://en.cppreference.com/w/cpp/filesystem/path

Ya, there is no + opperator for paths

Reply:
https://en.cppreference.com/w/cpp/filesystem/path/concat